public/index.html

```diff
@@ -118,14 +121,16 @@ <h5 v-for="movie in movies">{{ movie.Title }}</h5>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
         integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
         crossorigin="anonymous"></script>
+    <script src="https://apis.google.com/js/platform.js?onload=renderButton" async defer></script>
+
 
     <script src="./js/signin.js"></script>
     <script src="./js/index.js"></script>
     <script src="./js/omdb.js"></script>
     <script src="./js/components/vueHeader.js"></script>
     <script src="./js/components/vueFooter.js"></script>
-    <script src="https://apis.google.com/js/platform.js?onload=renderButton" async defer></script>
```

Comment:
Hey, is this a deletion?  I think my google button might depend on this for the auth.